static/app/components/events/featureFlags/featureFlagDistributions.tsx

```diff
@@ -0,0 +1,116 @@
+import {useCallback, useMemo} from 'react';
+import styled from '@emotion/styled';
+
+import useGroupFlags from 'sentry/components/events/featureFlags/useGroupFlags';
+import {OrderBy} from 'sentry/components/events/featureFlags/utils';
+import LoadingError from 'sentry/components/loadingError';
+import LoadingIndicator from 'sentry/components/loadingIndicator';
+import {t} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+import type {Group} from 'sentry/types/group';
+import {TagDistribution} from 'sentry/views/issueDetails/groupTags/tagDistribution';
+import type {GroupTag} from 'sentry/views/issueDetails/groupTags/useGroupTags';
+import {useEnvironmentsFromUrl} from 'sentry/views/issueDetails/utils';
+
+export default function FeatureFlagDistributions({
+  group,
+  search,
+  orderBy,
+}: {
+  group: Group;
+  orderBy: OrderBy;
+  search: string;
+}) {
+  const environments = useEnvironmentsFromUrl();
+
+  // Flags use the same endpoint and response format as tags, so we reuse TagDistribution, tag types, and "tag" in variable names.
+  const {
+    data = [],
+    isPending,
+    isError,
+    refetch,
+  } = useGroupFlags({
+    groupId: group.id,
+    environment: environments,
+  });
+
+  const tagValues = useMemo(
+    () =>
+      data.reduce<Record<string, string>>((valueMap, tag) => {
+        valueMap[tag.key] = tag.topValues.map(tv => tv.value).join(' ');
+        return valueMap;
+      }, {}),
+    [data]
+  );
+
+  const getSortedTags = useCallback(
+    (tags: readonly GroupTag[]) => {
```

Comment:
more of a nit and something to think about. But you could pull this outside of the react component and have two arguments instead of a useCallback